packages/cli/src/databases/entities/test-definition.ee.ts

```diff
@@ -5,7 +5,12 @@ import { AnnotationTagEntity } from '@/databases/entities/annotation-tag-entity.
 import type { TestMetric } from '@/databases/entities/test-metric.ee';
 import { WorkflowEntity } from '@/databases/entities/workflow-entity';
 
-import { WithTimestampsAndStringId } from './abstract-entity';
+import { jsonColumnType, WithTimestampsAndStringId } from './abstract-entity';
+
+// Entity representing a node in a workflow under test which is pinned for a test definition
+export type PinnedNodeItem = {
+	name: string;
```

Comment:
Did we go with names instead of IDs in the end? What about when someone updates the name of the node?

Reply:
How do we ensure that if the node is renamed, it's updated in the `TestDefinition` as well? I'd imagine we wouldn't have this issue if we used ids?

Reply:
We will switch to IDs before releasing this feature, but in a separate PR. This was left out to keep the PRs size manageable and to make the happy path working as soon as possible.